src/game_character.h

```diff
@@ -1083,6 +1086,14 @@ inline void Game_Character::IncAnimCount() {
 
 inline void Game_Character::IncAnimFrame() {
 	data()->anim_frame = (data()->anim_frame + 1) % 4;
+	SetAnimCount(0);
+}
+
+inline void Game_Character::ResetAnimation() {
+	SetAnimCount(0);
+	if (GetAnimationType() != RPG::EventPage::AnimType_fixed_graphic) {
+		SetAnimFrame(1);
```

Comment:
use the enum here RPG::EventPage::Frame_middle

Reply:
I can change these but we have 1 complication in that there are 2 frame middle values.

0 - left
1 - middle
2 - right
3 - middle

Reply:
I added `Frame_middle2` in https://github.com/EasyRPG/liblcf/pull/316

The logic in `sprite_character.cpp:62` can be changed to use that enum value when its merged.

Reply:
is in now :)